yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -172,3 +182,18 @@ func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*s
 	}
 	return nil
 }
+
+func (conv *DebeziumValueConverter) GetTableNameToSchema() map[string]map[string]map[string]string {
```

Comment:
(again for the future :D ) A probably nice way to deal with this might be: 
- instead of returning just a single columnType string for each column, return a column struct (that has all the information pertaining to that column - the name, the source type, the debezium type). 

Then,
- the valueConverter can work with that field struct (as opposed to a single keyword right now). 
- we can embed each column's schema in the data file descriptor (similar to how we add the delimiter/quote char/etc info in it. Then, we won't have to pass the schema directly to importBatch. 

Makes sense? Just add a TODO with some notes on how we can make this better later.